### PR TITLE
Chore/update mission deps

### PR DESCRIPTION
**Pull request by komret**
Update all Mission-related dependencies except Electron (updated recently by https://github.com/trezor/trezor-suite/pull/13795) and `tiny-secp256k1` (WIP in https://github.com/trezor/trezor-suite/pull/12261). Also, not realy sure what to do with `bcrypto`.

### CODE REVIEW

**Comment by komret** [comment]
I don't know why this breaks a mobile XRP test...

**Comment by vytick** [comment]
Looks like upgrade of jssha might have broken something related to XRP.
Screenshot 2024-08-23 at 16 33 38

**Comment by komret** [comment]
No idea how to fix that :( I reverted it back to the previous version. This dependency will be handled by @trezor/suite-connect team.

File: packages/utxo-lib/src/bip32.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@
 // - `fromBase58` and `toBase58` methods are using additional "network" param in bs58check.encode/decode (Decred support).
 
 import ecc from 'tiny-secp256k1';
-import wif from 'wif';
+import * as wif from 'wif';
```
Comment (Lemonexe) [inline packages/utxo-lib/src/bip32.ts:9]: :thought_balloon: Do we prefer to import everything from ES exports?
Why not `import { encode } from 'wif'`
or `import { encode as wifEncode } from 'wif'` to make the usage clearer?

Same with `bip66`..

**Review by Lemonexe — APPROVED** [review]
LGTM, CI is fine :heavy_check_mark:

**Pull request merged**